src/main/proto/envoy/config/core/v3/resolver.proto

```
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package envoy.config.core.v3;
+
+import "envoy/config/core/v3/address.proto";
```

Buf (1.54.0)

import "envoy/config/core/v3/address.proto": file does not exist

(COMPILE)